### PR TITLE
issue#47 add gNOI examples using gNOIc and gRPCurl

### PR DESCRIPTION
**Pull request by ksator**
Adding gNOI examples using gNOIc and gRPCurl

### CODE REVIEW

File: docs/examples/gnoi/gnoic/index.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+- Ping executes the ping command on the target and streams back the results
+- Traceroute executes the traceroute command on the target and streams back the results
+- As you can see in the proto file, the field VRF is not defined for these messages
+
+## About gNOI support on EOS
+
+https://eos.arista.com/eos-4-24-2f/gnoi/
+
+Examples:
```
Comment (fredhsu) [inline docs/examples/gnoi/gnoic/index.md:32]: Need a newline in between 32 and the list starting on 33 for the list to render correctly

File: docs/examples/gnoi/gnoic/index.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+## gNOIc overview
+
+gNOIc is a gNOI CLI client:
+
+- https://github.com/karimra/gnoic
+- https://gnoic.kmrd.dev/
+
+The following examples shows various gNOIc commands with Arista EOS devices.
+
+## Download && install gNOIc
```
Comment (fredhsu) [inline docs/examples/gnoi/gnoic/index.md:46]: Just need one ampersand :)

File: docs/examples/gnoi/gnoic/index.md (hunk 1 above)
Comment (ksator) [reply docs/examples/gnoi/gnoic/index.md:32]: done, thanks

File: docs/examples/gnoi/gnoic/index.md (hunk 2 above)
Comment (ksator) [reply docs/examples/gnoi/gnoic/index.md]: done, thanks

**Review by fredhsu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**